src/utils/arrow.rs

```diff
@@ -54,12 +53,18 @@ pub(crate) fn arrow_schema_from_column_names_and_range(
     column_names: &[String],
     row_idx: usize,
 ) -> Result<Schema> {
-    let mut fields = Vec::with_capacity(column_names.len());
-
-    for (col_idx, name) in column_names.iter().enumerate() {
-        let col_type = get_arrow_column_type(range, row_idx, col_idx)?;
-        fields.push(Field::new(&alias_for_name(name, &fields), col_type, true));
-    }
+    let fields = column_names
```

Comment:
I quite prefer your version! And it will probably be a bit faster. Even though it's just a small improvement, it remains an improvement :)

Reply:
alright, let me reopen it then !